web/app/themes/ppj/src/js/main.js

```diff
@@ -82,3 +86,17 @@ window.addEventListener('load', function() {
 
 
 }, false);
+
+/**
+ * function to close mobile nav menu if the screen
+ * is resized such that the desktop nav is now visible.
+ */
+window.addEventListener('resize', function() {
+  const screenWidth = window.ppj.getScreenWidth();
+
+  if (screenWidth >= 1024) {
+    window.ppj.closeNavMenu();
```

Comment:
There's a more performant way to achieve this. Whilst this works fine, it's going to be unnecessarily greedy on CPU cycles. Let's chat about it tomorrow to see if we think it's worth optimising.

Reply:
Done.